### DISCUSSION TITLE
Newbie question about config changes

### DISCUSSION THREAD

**Discussion by mirkh** [vufind-org/vufind#3947] [General]
Hi all,
We are recently starting our work on setting up vufind, where we are going to have several developers working on their own copy on the same server. 
Something I didn't find in the documentation is if all config changes are "loaded" as soon as they are made, or if you always need to reload apache after doing your change?
Hope that I will have a more educated question the next time I need to ask something. :)
/ Maria

**Comment by demiankatz** [comment]
Maria,

Changes to .ini files will take place immediately; you only need to restart Apache if you change the httpd-vufind.conf file.

Changes to the Laminas framework PHP-based configuration is cached if you are in production mode, but you can avoid this by switching to [development mode](https://vufind.org/wiki/development:troubleshooting#turn_on_development_mode).

In general, if something isn’t changing as expected, it never hurts to clear out the $VUFIND_LOCAL_DIR/cache directory.

I hope that helps you all get started, but don’t hesitate to reach out here or in Slack if you need more help!

**Reply by mirkh** [reply]
Hi, thanks a lot for confirming that no restarts are needed!
/ Maria

**Comment by demiankatz** [comment]
Glad I could help! I'm closing this discussion since the question is answered; of course, feel free to open new ones with future topics.
